tests/functional/test_uninstall.py

```diff
@@ -628,10 +628,6 @@ def test_uninstall_with_symlink(
     assert symlink_target.stat().st_mode == st_mode
 
 
-@pytest.mark.skipif(
-    "sys.version_info >= (3, 14)",
-    reason="Uninstall of .egg distributions not supported in Python 3.14+",
-)
```

Comment:
I initially thought this test was due to eggs not being detected with the importlib.metadata backend, but it turns out it's just a matter of the uninstallation order depending on the backend.